<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeView" name="nbtDataTreeView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_NBT_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionOpen_Minecraft_Save_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_in_Explorer"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionMoveUp"/>
    <addaction name="actionMoveDown"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindPrevious"/>
    <addaction name="actionFindNext"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_NBT_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionOpen_Folder"/>
   <addaction name="actionSave"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRename"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_ByteTag"/>
   <addaction name="actionAdd_ShortTag"/>
   <addaction name="actionAdd_IntTag"/>
   <addaction name="actionAdd_LongTag"/>
   <addaction name="actionAdd_FloatTag"/>
   <addaction name="actionAdd_DoubleTag"/>
   <addaction name="actionAdd_StringTag"/>
   <addaction name="actionAdd_ByteArrayTag"/>
   <addaction name="actionAdd_IntArrayTag"/>
   <addaction name="actionAdd_LongArrayTag"/>
   <addaction name="actionAdd_ListTag"/>
   <addaction name="actionAdd_CompoundTag"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionFindPrevious"/>
   <addaction name="actionFindNext"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/OpenFile.png</normaloff>:/icons/16x16/OpenFile.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Folder.png</normaloff>:/icons/16x16/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Exit.png</normaloff>:/icons/16x16/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen_Minecraft_Save_Folder">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Folder.png</normaloff>:/icons/16x16/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Minecraft Save Folder</string>
   </property>
  </action>
  <action name="actionOpen_in_Explorer">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Folder.png</normaloff>:/icons/16x16/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open in Explorer</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Save.png</normaloff>:/icons/16x16/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Cut.png</normaloff>:/icons/16x16/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Copy.png</normaloff>:/icons/16x16/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Paste.png</normaloff>:/icons/16x16/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Rename.png</normaloff>:/icons/16x16/Rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Edit.png</normaloff>:/icons/16x16/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Delete.png</normaloff>:/icons/16x16/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAdd_ByteTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/ByteTag.png</normaloff>:/icons/16x16/ByteTag.png</iconset>
   </property>
   <property name="text">
    <string>Add ByteTag</string>
   </property>
  </action>
  <action name="actionAdd_ShortTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/ShortTag.png</normaloff>:/icons/16x16/ShortTag.png</iconset>
   </property>
   <property name="text">
    <string>Add ShortTag</string>
   </property>
  </action>
  <action name="actionAdd_IntTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/IntTag.png</normaloff>:/icons/16x16/IntTag.png</iconset>
   </property>
   <property name="text">
    <string>Add IntTag</string>
   </property>
  </action>
  <action name="actionAdd_LongTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/LongTag.png</normaloff>:/icons/16x16/LongTag.png</iconset>
   </property>
   <property name="text">
    <string>Add LongTag</string>
   </property>
  </action>
  <action name="actionAdd_FloatTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/FloatTag.png</normaloff>:/icons/16x16/FloatTag.png</iconset>
   </property>
   <property name="text">
    <string>Add FloatTag</string>
   </property>
  </action>
  <action name="actionAdd_DoubleTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/DoubleTag.png</normaloff>:/icons/16x16/DoubleTag.png</iconset>
   </property>
   <property name="text">
    <string>Add DoubleTag</string>
   </property>
  </action>
  <action name="actionAdd_StringTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/StringTag.png</normaloff>:/icons/16x16/StringTag.png</iconset>
   </property>
   <property name="text">
    <string>Add StringTag</string>
   </property>
  </action>
  <action name="actionAdd_ByteArrayTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/ByteArrayTag.png</normaloff>:/icons/16x16/ByteArrayTag.png</iconset>
   </property>
   <property name="text">
    <string>Add ByteArrayTag</string>
   </property>
  </action>
  <action name="actionAdd_IntArrayTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/IntArrayTag.png</normaloff>:/icons/16x16/IntArrayTag.png</iconset>
   </property>
   <property name="text">
    <string>Add IntArrayTag</string>
   </property>
  </action>
  <action name="actionAdd_LongArrayTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/LongArrayTag.png</normaloff>:/icons/16x16/LongArrayTag.png</iconset>
   </property>
   <property name="text">
    <string>Add LongArrayTag</string>
   </property>
  </action>
  <action name="actionAdd_ListTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/ListTag.png</normaloff>:/icons/16x16/ListTag.png</iconset>
   </property>
   <property name="text">
    <string>Add ListTag</string>
   </property>
  </action>
  <action name="actionAdd_CompoundTag">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/CompoundTag.png</normaloff>:/icons/16x16/CompoundTag.png</iconset>
   </property>
   <property name="text">
    <string>Add CompoundTag</string>
   </property>
  </action>
  <action name="actionNew_NBT_File">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/NewNbtFile.png</normaloff>:/icons/16x16/NewNbtFile.png</iconset>
   </property>
   <property name="text">
    <string>New NBT File</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Save.png</normaloff>:/icons/16x16/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Save.png</normaloff>:/icons/16x16/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Refresh.png</normaloff>:/icons/16x16/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/MoveUp.png</normaloff>:/icons/16x16/MoveUp.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/MoveDown.png</normaloff>:/icons/16x16/MoveDown.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/About.png</normaloff>:/icons/16x16/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/Find.png</normaloff>:/icons/16x16/Find.png</iconset>
   </property>
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/FindNext.png</normaloff>:/icons/16x16/FindNext.png</iconset>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="toolTip">
    <string>Find Next</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionFindPrevious">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/16x16/FindPrev.png</normaloff>:/icons/16x16/FindPrev.png</iconset>
   </property>
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="toolTip">
    <string>Find Previous</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
